ngraph/cmake/coverage.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(NGRAPH_COVERAGE_GCDA_DATA_DIRECTORY "${CMAKE_BINARY_DIR}/ngraph")
```

Comment:
let's have common directory for GCDA data for the whole OpenVINO

Reply:
Fixed